src/PrestaShopBundle/Command/ConfigCommand.php

```diff
@@ -0,0 +1,320 @@
+        if (!is_null($this->shopConstraint->getShopId())) {
+            $onlyShopId = $this->shopConstraint->getShopId()->getValue();
+        }
+        $languages = $this->languageDataProvider->getLanguages($onlyActive, $onlyShopId);
+
+        $found = current(array_filter($languages, function (array $item) use ($inputlang) {
+            $key = is_numeric($inputlang) ? 'id_lang' : 'iso_code';
+
+            return isset($item[$key]) && $inputlang == $item[$key];
+        }));
+
+        if (!$found) {
+            throw new Exception('Invalid language', self::STATUS_INVALID_OPTIONS);
+        }
+
+        $this->idLang = (int) $found['id_lang'];
+    }
+
+    /**
+     * Get and print out one configuration value
+     */
+    private function get(): void
+    {
+        $key = $this->input->getArgument('key');
+
+        if (strpos($key, '*') !== false) {
+            if (!$this->idLang) {
+                throw new Exception('Using wildcards requires --lang option', self::STATUS_LANG_REQUIRED);
+            }
+            try {
+                $keys = array_filter($this->configuration->keys(), function (string $v) use ($key) {
+                    return fnmatch($key, $v);
+                });
+            } catch (NotImplementedException $e) {
+                throw new Exception('Not implemented yet', self::STATUS_NOT_IMPLEMENTED);
+            } catch (Exception $e) {
+                $msg = sprintf('Could not load all configuration keys: %s', $e->getMessage());
+                throw new Exception($msg, self::STATUS_ERROR);
+            }
+        } else {
+            $keys = [$key];
+        }
+
+        foreach ($keys as $key) {
+            $value = $this->configuration->get($key, null, $this->shopConstraint);
+
+            // non language values will be just strings
+            // but for language dependant values the response is an array
+            if (is_array($value)) {
+                if (!$this->idLang) {
+                    $msg = sprintf('%s is language dependant, --lang option is required', $key);
+                    throw new Exception($msg, self::STATUS_LANG_REQUIRED);
+                }
+                $value = $value[$this->idLang];
+            }
+
+            $this->io->success(sprintf('%s=%s', $key, $value));
+        }
+    }
+
+    /**
+     * Set and print out one configuration value
+     */
+    private function set(): void
+    {
+        $key = $this->input->getArgument('key');
+
+        if (strpos($key, '*') !== false) {
+            throw new Exception('Set action does not support wildcards', self::STATUS_INVALID_OPTIONS);
+        }
+
+        $newvalue = $this->input->getOption('value');
+
+        // new value is required for set
+        if (is_null($newvalue)) {
+            throw new Exception('Value required for action "set"', self::STATUS_VALUE_REQUIRED);
+        }
+
+        // make the value language array if lang is set
+        if (!is_null($this->idLang)) {
```

Comment:
```suggestion
        if (false !== $this->idLang) {
```

Never use is_null it's really really bad for performance